extensions/transform/org.eclipse.smarthome.transform.regex/README.md

```diff
@@ -11,7 +11,7 @@ The regular expression in the format `s/<regex>/result/` (without `g`), replaces
 If the regular expression contains a [capture group](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#cg) defined by `()`, it returns the captured string. 
 Multiple capture groups can be used to retrieve multiple strings and can be combined as a result string defined in the `substitution`.
 
-The transformation can be set to be restricted to only match if the input string beginns with an character by prepending `^` to the beginning of a pattern or to only match if the input string ends with a specified charactger by appending `$` at the end.
+The transformation can be set to be restricted to only match if the input string beginns with an character by prepending `^` to the beginning of a pattern or to only match if the input string ends with a specified character by appending `$` at the end.
```

Comment:
"string beginns with an character" -> "string begins with a character"

Reply:
good catch, do you mind fixing them?